lib/ex_sdp/attribute/fmtp.ex

```diff
@@ -234,7 +236,8 @@ defmodule ExSDP.Attribute.FMTP do
                other -> other
              end
            end) do
-      {rest, Map.put(fmtp, :redundant_payloads, redundant_payloads)}
+      # We need uniq because Chrome sends 111/111 for Opus ¯\_(ツ)_/¯
```

Comment:
You're right, this makes our parsing valid, otherwise, we'd have to check `rtpmapping` to determine the meaning of `66` here or put it under both fields.